lib/main.coffee

```diff
@@ -7,38 +7,75 @@ module.exports =
 
   config:
     compatMode:
+      title: 'Compatibility mode (AsciiDoc Python)'
       type: 'boolean'
-      default: true
+      default: false
+      order: 1
+    forceExperimental:
+      title: 'Force enable experimental extensions'
+      description: '''
+        The features behind this attribute are subject to change and may even be removed in a future version.
+
+        Currently enables the UI macros (`button`, `menu` and `kbd`).
+        '''
+      type: 'boolean'
+      default: false
+      order: 2
     showTitle:
+      description: '''
+        If set, displays an embedded document’s title.
+
+        Mutually exclusive with the notitle attribute.
+        '''
       type: 'boolean'
       default: true
+      order: 3
     safeMode:
+      description: '''
+        Set safe mode level: `unsafe`, `safe`, `server` or `secure`.
+
+        Disables potentially dangerous macros in source files, such as `include::[]`.
```

Comment:
In the next version of Atom, we will able to define a description for each value:

```coffee

    safeMode:
      description: '''
        Set safe mode level: `unsafe`, `safe`, `server` or `secure`.

        Disables potentially dangerous macros in source files, such as `include::[]`.

        http://asciidoctor.org/docs/user-manual/#running-asciidoctor-securely
        '''
      type: 'string'
      default: 'safe'
      enum: [
        value: 'unsafe',
        description: 'A safe mode level that disables any security features enforced by Asciidoctor.'
      ,
        value: 'safe',
        description: 'This safe mode level prevents access to files which reside outside of the parent directory of the source file. '
      ,
        value: 'server',
        description: 'A safe mode level that disallows the document from setting attributes that would affect the rendering of the document.'
      ,
        value: 'secure',
        description: 'A safe mode level that disallows the document from attempting to read files from the file system and including their contents into the document.'
      ]
```